src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -51,15 +126,16 @@ public static final String getHelpSummary() {
                 .append("Delete an inventory item: delete-i INDEX\n")
                 .append("Edit an inventory item: edit-i INDEX [n/NAME] [q/QUANTITY]"
                         + " [s/SUPPLIER] [max/MAX_QUANTITY] [t/TAG]\n")
-                .append("Find an inventory item: find-i KEYWORD [MORE_KEYWORDS]\n")
+                .append("Find an inventory item: find-i n/KEYWORD [MORE_KEYWORDS]\n")
                 .append("Clear all inventory items: clear-i\n")
                 .append("List all inventory items: list-i\n")
+                .append(DIVIDER)
                 .append("DELIVERY SPECIFIC COMMANDS\n")
                 .append(DIVIDER)
                 .append("Add delivery: add-d n/NAME p/PHONE a/ADDRESS o/ORDER\n")
                 .append("Delete a delivery: delete-d INDEX\n")
                 .append("Edit a delivery: edit-d INDEX [n/NAME] [p/PHONE] [a/ADDRESS] [o/ORDER]\n")
-                .append("Find a delivery: find-d KEYWORD [MORE_KEYWORDS]\n")
+                .append("Find a delivery: find-d n/KEYWORD [MORE_KEYWORDS]\n")
```

Comment:
this implies that `n/` is mandatory. 
Maybe something like `find-d [n/NAME | p/PHONE | a/ADDRESS | o/ORDER]`